.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: test
+on: [push, pull_request]
+jobs:
+  ubuntu-2004-chromium:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          # need this to also fetch tags
+          fetch-depth: 0
+
+      - name: Build release tarball
+        run: make dist-gzip
+
+      - name: Build/install cockpit-podman package and dependencies
+        run: |
+          cp cockpit-podman-*.tar.gz /var/tmp/
+          cp -r packaging/debian/ /var/tmp
+          sudo -n env APT_INSTALL_OPTIONS="-t focal-backports" test/vm.install
+
+      - name: Install test dependencies
+        run: |
+          sudo -n apt-get install -y chromium-browser python3-libvirt
+          make test/common bots
+
+      # interactive debugging
+      # - uses: mxschmitt/action-tmate@v3
+
+      - name: Run test
+        # FIXME: debian-testing is not quite accurate, but our tests don't recognize ubuntu yet
+        run: TEST_AUDIT_NO_SELINUX=1 TEST_OS=debian-testing test/common/run-tests -tv --machine 127.0.0.1:22 --browser 127.0.0.1:9090
```

Comment:
We may want to drop `-tv` for the final version to make the logs less massive.

Reply:
Yeah, I think `-v` is enough. `-t` for normal runs is just too much.